lumen/bootstrap/app.php

```diff
@@ -79,6 +79,9 @@
 */
 
 $app->register(App\Providers\AppServiceProvider::class);
+$app->register(Illuminate\Redis\RedisServiceProvider::class);
```

Comment:
Define the custom and external service providers into `the AppServiceProvider` is a good practice, but don't worry both works 😆